fishjam/media_events/server/server.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+package fishjam.media_events.server;
+
+import "fishjam/media_events/shared.proto";
+
+// Defines any type of message sent from Membrane RTC Engine to Peer
+message MediaEvent {
+  // SCHEMAS
+  message SimulcastConfig {
+    bool enabled = 1;
+    repeated media_events.Encoding active_encodings = 2;
+    repeated media_events.Encoding disabled_encodings = 3;
+  }
+
+  message Track {
+    string track_id = 1;
+    media_events.Metadata metadata = 2;
+    SimulcastConfig simulcast_config = 3;
```

Comment:
Because this PR is titled `Media events no simulcast`, it is appropriate to exclude `SimulcastConfig` from `Track`. Since protobufs should be forward and backward compatible, we can add it later. This gives us more time to refine the structure of simulcast events

Reply:
I asked about it @roznawsk.